src/main/kotlin/org/jetbrains/plugins/bsp/magicmetamodel/impl/workspacemodel/impl/updaters/transformers/ModuleDetailsToGoModuleTransformer.kt

```diff
@@ -11,48 +11,51 @@ import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.GoModuleDepe
 import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.ModuleDetails
 import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.toBsp4JTargetIdentifier
 import java.net.URI
-import java.nio.file.Path
 import kotlin.io.path.toPath
 
 internal class ModuleDetailsToGoModuleTransformer(
-    private val targetsMap: Map<BuildTargetId, BuildTargetInfo>,
-    private val projectDetails: ProjectDetails,
-    moduleNameProvider: ModuleNameProvider,
-    projectBasePath: Path,
+  private val targetsMap: Map<BuildTargetId, BuildTargetInfo>,
+  private val projectDetails: ProjectDetails,
+  moduleNameProvider: ModuleNameProvider,
 ) : ModuleDetailsToModuleTransformer<GoModule>(targetsMap, moduleNameProvider) {
-    override val type = "GO_MODULE"
+  override val type = "GO_MODULE"
 
-    override fun transform(inputEntity: ModuleDetails): GoModule {
-        val goBuildInfo = extractGoBuildTarget(inputEntity.target) ?: error("extract nie działa")
+  override fun transform(inputEntity: ModuleDetails): GoModule {
+    val goBuildInfo = extractGoBuildTarget(inputEntity.target) ?: error("Transform error, cannot extract GoBuildTarget")
```

Comment:
```suggestion
```

Reply:
The thing is, extractGoBuildTarget can return GoBuildTarget? and the question mark there makes sense, as in all the other languages. I think when using go transformer on a target that is not of a "go" kind we should throw an error, as this should never happen in our usage

Reply:
to juz tam nizej napisalem